bartop-io/bartop
# Integrate Auth0

Migration of this https://github.com/dpopp07/bartop/pull/52

We had to migrate b/c Travis doesn't trust builds that don't originate from the repo. We're going to have to figure out adjusting testing for PRs so that people can still make OS contributions that pass Travis.

Here's the old jawn:

- [x]  add auth0
- [x] run auth object through context
- [x] run auth through state
- [x] run auth state through local storage

To test:
1. Log in :)
2. Examine redux state (https://github.com/zalmoxisus/redux-devtools-extension) and verify it holds expected information (accessToken, expiresIn, user information)
